14_functions_lab/01.py

```diff
@@ -4,4 +4,30 @@
     mymul - returns the multiplication of its input arguments
     Ignore non-numeric arguments
 """
+import operator 
 
+def mysum(*paramsin):
+    res = 0
+    res = sum([int(val) for val in paramsin if type(val) is int])
+    
+    return res
+
+def mymul(*paramsin):   
+    if len(paramsin)>0:
```

Comment:
Since *1 is a noop, you can just append a fake 1 to the end of the list:

```
reduce(operator.mul,[int(val) for val in paramsin + [1] if type(val) is int])
```